stubs/docutils/docutils/transforms/__init__.pyi

```diff
@@ -1,14 +1,31 @@
-from _typeshed import Incomplete
+from collections.abc import Iterable
+from types import ModuleType
+from typing import Any
 
-from docutils.nodes import Node, document
+from docutils import ApplicationError, Component, TransformSpec, nodes
+from docutils.nodes import Node
+
+class TransformError(ApplicationError): ...
 
 class Transform:
-    def __init__(self, document: document, startnode: Node | None = None): ...
-    def __getattr__(self, name: str, /) -> Incomplete: ...
+    default_priority: int | None
+    document: nodes.document
+    startnode: Node | None
+    language: ModuleType
+    def __init__(self, document: nodes.document, startnode: Node | None = None) -> None: ...
+    def apply(self, **kwargs: Any) -> None: ...
 
-class Transformer:
-    def __init__(self, document: document): ...
+class Transformer(TransformSpec):
+    transforms: list[tuple[str, type[Transform], Node | None]]
+    document: nodes.document
+    applied: list[tuple[int, type[Transform], nodes.pending, Any]]
```

Comment:
```suggestion
    applied: list[tuple[int, type[Transform], nodes.pending | None, Any]]
```

Reply:
Here, you really want to allow None nodes (that's their "default" values when adding a new transformation).